clang/lib/DPCT/SaveNewFiles.cpp

```diff
@@ -395,6 +396,11 @@ void applyPatternRewriterToCmakeScriptFile(const std::string &InputString,
 
   // Convert cmake command to lower case in cmake script files
   LineEndingString = convertCmakeCommandsToLower(LineEndingString);
+
+  std::map<std::string, std::string> VariablesMap;
```

Comment:
please put all the logic into MigrateCmakeScript.cpp and expose an interface.

Reply:
Okay